src/app/create/components/CategoryTree.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { TriangleAlertIcon } from 'lucide-react';
+import Cascader, { CascaderProps } from 'rc-cascader';
+
+import useGetCategory from '@/apis/queryHooks/category/useGetCategory';
+import { TransformCategoriesToOptions } from '@/apis/types/category';
+
+import * as S from '../Basicauction.css';
+import './CategoryTree.css';
+import { AuctionInputs } from '../types/InputTypes';
+
+function CategoryTree() {
+  const {
+    control,
+    formState: { errors },
+  } = useFormContext<AuctionInputs>();
+
+  const { data } = useGetCategory({ categoryType: 'create' });
+
+  const [inputValue, setInputValue] = useState('');
+
+  const onChange: CascaderProps<TransformCategoriesToOptions>['onChange'] = (
+    _,
+    selectedOptions,
+  ) => {
+    const lastSelectedValue = Number(selectedOptions[selectedOptions.length - 1]?.value);
+    setInputValue(selectedOptions.map(o => o.label).join(' > '));
+    return lastSelectedValue;
+  };
```

Comment:
hookform의 `setValue` 사용하면 useState 필요 없을 것 같네요!

```suggestion
    setValue('categoryIdsRequired', [lastSelectedValue]);
    const categoryLabels = selectedOptions.map(option => option.label).join(' > ');

    return categoryLabels;
  };
```